pepper-apis/configure.rb

```diff
@@ -171,24 +181,31 @@ def render_from_path(path, output_file_name = nil)
     end
   end
   copy_file_from_path(path)
-  docker_cmd = [
-      "docker", "run", "--rm", "-w", "/w", "-v", "#{Dir.pwd}:/w",
-      "-e", "VAULT_TOKEN=#{$vault_token}", "-e", "ENVIRONMENT=#{$env}", "-e", "VERSION=#{$version}",
-      "-e", "BUILD_CONTAINERS=#{$build_containers_flag}",
-      "-e", "DOCS_PROXIED_HOST=#{$docsProxiedHost}",
-      "-e", "NGINX_PROXIED_HOST=#{$nginxProxiedHost}",
-      "-e", "NO_SYSLOG=#{$noSyslog}",
-      "-e", "DEBUG=#{$debug_flag}",
-      "-e", "DIR=#{$dir}", "-e", "IMAGE=#{$image}",
-      "-e", "STUDY_GUID=#{$study_guid}",
-      "-e", "STUDY_KEY=#{$study_key}",
-      "-e", "IMAGE_NAME=#{$image_base}",
-      "broadinstitute/dsde-toolbox:latest",
-      "consul-template", "-config=/etc/consul-template/config/config.json",
-      "-template=#{file_name}:#{output_file_name}",
-      "-once"
-  ]
-  Open3.popen3(*docker_cmd) { |stdin, stdout, stderr, wait_thread|
+  if $use_docker == true
+    vault_cmd = [
+             "docker", "run", "--rm", "-w", "/w", "-v", "#{Dir.pwd}:/w",
+                  "-e", "VAULT_TOKEN=#{$vault_token}", "-e", "ENVIRONMENT=#{$env}", "-e", "VERSION=#{$version}",
+                  "-e", "GAE=#{$gae}",
+                  "-e", "BUILD_CONTAINERS=#{$build_containers_flag}",
+                  "-e", "DOCS_PROXIED_HOST=#{$docsProxiedHost}",
+                  "-e", "NGINX_PROXIED_HOST=#{$nginxProxiedHost}",
+                  "-e", "NO_SYSLOG=#{$noSyslog}",
+                  "-e", "DEBUG=#{$debug_flag}",
+                  "-e", "DIR=#{$dir}", "-e", "IMAGE=#{$image}",
+                  "-e", "STUDY_GUID=#{$study_guid}",
+                  "-e", "STUDY_KEY=#{$study_key}",
+                  "-e", "IMAGE_NAME=#{$image_base}",
+                  $dsde_toolbox_image_name,
+                  "consul-template", "-config=/etc/consul-template/config/config.json",
+                  "-template=#{file_name}:#{output_file_name}",
+                  "-once"
+    ]
+  else
+    vault_cmd = ["consul-template", "-config=/etc/consul-template/config/config.json",
+                              "-template=#{file_name}:#{output_file_name}", "-once"]
+  end
+
```

Comment:
Allow for possibility of having consul-template and vault installed locally. This is what happens in CircleCi docker image (calling out to from from Docker in CircleCi was complicated).